Library/ViewModels/ProjectPamphletContentViewModelTests.swift

```diff
@@ -539,4 +622,78 @@ final class ProjectPamphletContentViewModelTests: TestCase {
       self.loadMinimalProjectIntoDataSource.assertValues([])
     }
   }
+
+  func testLoadProjectIntoDataSource_CreatorDetailsFailure_ExperimentalVariant() {
+    let project = Project.template
+
+    let mockService = MockService(fetchProjectCreatorDetailsResult: .failure(.invalidInput))
+
+    let optimizelyClient = MockOptimizelyClient()
+      |> \.experiments .~ [
+        OptimizelyExperiment.Key.nativeProjectPageConversionCreatorDetails.rawValue: OptimizelyExperiment.Variant.variant1.rawValue
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/kickstarter/ios-oss/blob/8676c26a01ecffa0f982d0e5b09f2191cd795621/Library/ViewModels/ProjectPamphletContentViewModelTests.swift#L633"></span>Line should be 110 characters or less: currently 131 characters<br />
<code>line_length</code> <code>ProjectPamphletContentViewModelTests.swift:633</code></td>
    </tr>
  </tbody>
</table>